services/github/github-stars.service.js

```diff
@@ -34,8 +35,8 @@ module.exports = class GithubStars extends LegacyService {
           }
           try {
             badgeData.text[1] = metric(JSON.parse(buffer).stargazers_count)
-            badgeData.colorscheme = null
-            badgeData.colorB = '#4183C4'
+            badgeData.colorscheme = 'blue'
+            setBadgeColor(badgeData, data.colorB)
```

Comment:
Please remove this line (`setBadgeColor(badgeData, data.colorB)`). It causes a problem when you do not override `colorB`. In this case `data.colorB` is not defined and `setBadgeColor` sets color to `red` 
https://github.com/badges/shields/blob/master/lib/badge-data.js#L64

Reply:
> Please remove this line (`setBadgeColor(badgeData, data.colorB)`). 

I'm not sure that's the way to fix this. Maybe I shouldn't set `.colorscheme` directly because `setBadgeColor()` will actually do that for me. I think the first version might be a better way. And I think to simplified the code I should remove `badgeData.colorscheme = null`.
```diff
-            badgeData.colorscheme = null
-            badgeData.colorB = '#4183C4'
+            setBadgeColor(badgeData, data.colorB || 'blue')
```

Reply:
😓 I just realize I've misunderstood something. It's ashamed. Sorry.